### ISSUE TITLE
[Bug]: You are not authorized to edit themes

### ISSUE THREAD

**Issue by ebro** [Shopify/shopify-cli#2231]
### Please confirm that you have:

- [X] Searched [existing issues](https://github.com/Shopify/shopify-cli/issues) to see if your issue is a duplicate. (If you’ve found a duplicate issue, feel free to add additional information in a comment on it.)
- [X] Reproduced the issue in [the latest CLI version](https://github.com/Shopify/shopify-cli/releases).

### What type of project are you working on?

theme

### Expected behavior

I have a partner account and there I have created a store as the owner. If I now log in via shopify-cli and trigger e.g. shopify theme pull, I get an error message regarding the authorization.

I assume that if I have created the store, I should also have the correct permissions to edit the store.

I can't give myself special permissions either.

### Actual behavior

 Error 
┃ You are not authorized to edit themes on ...

Try this 
┃ Check if your user is activated, has permission to edit themes at the store, and try to re-login.

### Reproduction steps

1. shopify login --store xy
2. shopify theme pull

### Operating System

Mac OS Monterey

### CLI version (run `shopify version` if you're not sure)

2.15.3

### Ruby version (run `ruby -v` if you're not sure)

2.6.8p205

**Comment by ebro** [comment]
One more input. 
I don't even have permission to log out of the cli.

`
shopify logout  

> Error 
 You are not authorized to edit themes on ebro-dev-shop.myshopify.com.

> Try this
 Check if your user is activated, has permission to edit themes at the store, and try to re-login.

`

**Comment by ebro** [comment]
I was now able to solve my problem.

1. deleted in the folder ~/.cache the folder shopify
2. logged out from everything in my main browser (safari). Deleted all cookies from shopify.
3. opened another browser (Chrome) and logged into the partner account. I had to reset the password for some reason.

After that I was able to log into the shopify-cli again and now I have all the necessary permissions again.
